src/styles/main.scss

```diff
@@ -1,3 +1,238 @@
-body {
+$color-main-accent: #060b35;
+
+$color-secondary: #616070;
+
+$color-accent-blue: #00acdc;
+
+* {
+  box-sizing: border-box;
+}
+
+//========== Body, Section, Row =========== //
+.body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+}
+
+.catalog {
+  display: flex;
+  flex-wrap: wrap;
+  width: 1024px;
+  margin: 0 auto;
+  padding: 50px 40px;
+}
+
+//========== Header ============= //
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.header__logo {
+  display: flex;
+  height: 40px;
+  width: 40px;
+  cursor: pointer;
+  align-self: center;
+}
+
+.nav {
+  display: flex;
+}
+
+.nav__list {
+  margin: 0;
+  padding: 0;
+  display: flex;
+  list-style: none;
+  text-transform: uppercase;
+}
+
+.nav__item {
+  display: flex;
+  height: 60px;
+}
+
+.nav__item:not(:last-child) {
+  margin-right: 20px;
+}
+
+.nav__link {
+  padding-top: 22.5px;
+  padding-bottom: 22.5px;
+  position: relative;
+  align-content: center;
+  align-items: center;
+  display: flex;
+  color: #{$color-main-accent};
+  font-weight: 500;
+  font-size: 12px;
+  text-decoration: none;
+}
+
+.nav__link:hover {
+  color: #{$color-accent-blue};
+}
+
+.is-active {
+  position: relative;
+  justify-self: center;
+  color: #{$color-accent-blue};
+  font-weight: 500;
+  font-size: 12px;
+  text-decoration: none;
+  padding-top: 22.5px;
+  padding-bottom: 22.5px;
+  align-content: center;
+  align-items: center;
+  display: flex;
+}
+
+.is-active::after {
+  content: "";
+  position: absolute;
+  width: 100%;
+  height: 4px;
+  top: 56px;
+  left: 0;
+  background: #{$color-accent-blue};
+  border-radius: 8px;
+  display: block;
+}
+
+// ============= Card ============= //
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  background: #ffff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+}
+
+.card--distance {
+  margin-right: 48px;
+  margin-bottom: 46px;
+}
+
+.card--distance:nth-child(4n) {
+  margin-right: 0;
+}
+
+.card--distance:nth-last-child(-n + 4) {
+  margin-bottom: 0;
+}
+
+.card__img {
+  display: block;
+  margin: 0 auto 40px;
+  width: 160px;
+  height: 134px;
+}
+
+.card__title {
+  font-size: 12px;
+  font-weight: 500;
+  color: #{$color-main-accent};
+  line-height: 18px;
+  margin: 0 0 4px;
```

Comment:
don't repeat yourself. no need to always set all margins.
try to set only `margin-bottom`, for other you have paddings for `card`

Reply:
don't forget about css properties order
http://codeguide.academy/html-css.html#css-order

Reply:
> don't repeat yourself. no need to always set all margins. try to set only `margin-bottom`, for other you have paddings for `card`

here we need to set all margins